### ISSUE TITLE
Design:  OSO user with existing projects import path

### ISSUE THREAD

**Issue by catrobson** [fabric8-ui/fabric8-ux#687]
If a user has an OSO account with existing projects, we should allow the user to import these into OSIO in a clean and understandable way.  Users could also continue to add 'non-OSIO' projects into OSO, and we should handle these as well.

Verification Criteria:
* User flows created for the various paths that can occur
* Designs captured for each flow and shared on InVision, Fabric8 wiki page

**Comment by lwrigh** [comment]
I'm started to put together a couple of flows where the user is importing existing OSO projects in OSIO: https://redhat.invisionapp.com/share/FRDRC1DED any feedback would be much appreciated @catrobson @bdellasc @Essjaysee @mceledonia

**Comment by kbsingh** [comment]
just a note here : here is no expectation on migration into osio from oso - due to the way osio works, it is highly unlikely we will ever be able to import an existing app from oso without major rework being done.

**Comment by maxandersen** [comment]
Yes, to @kbsingh 's point importing existing openshift namespace/projects doesn't really make sense. The initial issue is we need a way to *make room* for the namespace that hosts openshift io tenants and if user already have something else there AND are on the free plan neither they or we will be able to create that new namepsace.

**Comment by catrobson** [comment]
I'm not sure I'm following the initial use case fully @kbsingh @maxandersen. To make space for the OpenShift.io tenant would this need to happen when I sign up for OSIO (before I ever see the OSIO console), or upon any 'project' creation that is going to actually ask for tenant space to stand up che, jenkins, and environments?  What are the calls to action that we're expecting to be able to provide to the user in this situation?

By the way "it is highly unlikely we will ever be able to import an existing app from oso without major rework being done" is a big issue in my mind.  I'd be fine if we have to send the user through the wizard for each project, but having no future import path is a problem we should make sure PM is aware of as I don't think this is well understood.

**Comment by catrobson** [comment]
@lwrigh this was discussed during our morning integration meeting.  The immediate need is that users are coming into OpenShift.io with existing projects in OpenShift Online, and they are only allowed 1 namespace in the "starter" free tier.  This means that any user trying to set up something in OpenShift.io in this situation will be unable to successfully.

Therefore, our first design need is to build something into the UI that will alert the user if the namespace is used by OSO so they can know to go clean out their OSO projects in order to successfully use OSIO.   We likely need this to be present on the header that there is a problem, and can then expand by user click, or automatically expand on first log in scenario, so the user can clearly see the problem and get a link to the OSO console to resolve it.  Can we prioritize this use case and design over the user flows we were working through above?

**Comment by lwrigh** [comment]
@catrobson I can focus on this use case scenario and put the other flow on the back burner for now
